### PR TITLE
Allowing to get elements attributes from a list of ids, even if some elements are not found

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/test/java/org/gridsuite/directory/server/DirectoryTest.java [hunk 1]
```diff
@@ -528,7 +528,14 @@ public void testGetElement() {
         ElementAttributes scriptAttributes = toElementAttributes(UUID.randomUUID(), "Script", FILTER, false, "user1");
         insertAndCheckSubElement(rootDirUuid, false, scriptAttributes);
 
-        var res = getElements(List.of(contingencyAttributes.getElementUuid(), filterAttributes.getElementUuid(), scriptAttributes.getElementUuid()), "user1");
+        var res = getElements(List.of(contingencyAttributes.getElementUuid(), filterAttributes.getElementUuid(), UUID.randomUUID()), "user1", true, 404);
+        assertTrue(res.isEmpty());
+
+        res = getElements(List.of(contingencyAttributes.getElementUuid(), filterAttributes.getElementUuid(), UUID.randomUUID()), "user1", false, 200);
```
Comment (sBouzols) [inline src/test/java/org/gridsuite/directory/server/DirectoryTest.java:534]: We should expect a HTTP 206 code for a 'Partial Content' return code ?

**Review by sBouzols — COMMENTED** [review]
Code Review ok
Can't test
We should respect HTTP conventions. Chould we use 206 code for 'partial Content' ?

File: src/test/java/org/gridsuite/directory/server/DirectoryTest.java (hunk 1 above)
Comment (sBouzols) [reply src/test/java/org/gridsuite/directory/server/DirectoryTest.java:534]: No it doesn't seems to be the 206 code which correspond...

Comment (sBouzols) [reply src/test/java/org/gridsuite/directory/server/DirectoryTest.java:524]: 207 is for WebDav HTTP extension and need an xml body, it doesn't fit either.
Ok for 200 return code then, but we always need tp check if the result of this Get method is complete which is a pain.

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**